unordered-containers.cabal

```diff
@@ -56,7 +56,8 @@ library
   build-depends:
     base >= 4.9 && < 5,
     deepseq >= 1.1,
-    hashable >= 1.0.1.1 && < 1.5
+    hashable >= 1.0.1.1 && < 1.5,
+    template-haskell
```

Comment:
`template-haskell` dependency without a bound is risky (as you can see yourself even `Lift` had changed!)

Reply:
Yeah, I'll put one in. In practice, the `base` version determines it to within a sufficiently tight bound, but it can't hurt to specify.

Reply:
Except `base <5`, i.e. essentially without upper bound